runtime-libraries/bom/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>com.ibm.bamoe</groupId>
     <artifactId>bamoe-runtime-libraries</artifactId>
-    <version>9.0-SNAPSHOT</version>
+    <version>9.0.1-SNAPSHOT</version>
```

Comment:
I'm not sure if using 9.0.1-SNAPSHOT is needed, usually just use 9.0-SNAPSHOT is enough

Reply:
Fixed.